notebook2/notebook/static/styles/variables/typeography.scss

```diff
@@ -0,0 +1,3 @@
+$weight-medium: 500;
+$weight-heavy: 700;
+$weight-light: 300;
```

Comment:
I don't see these used anywhere, except for medium. I guess future PRs will use the other ones? Also, what's the thought on using `500` versus `normal` (defined as `400`)? On my laptop, I see no difference between Open Sans at 500 and Open Sans at 400. Are `normal` and `bold` sufficient for our purposes?

Reply:
It used to be the case that browsers would have different ideas of what “normal” meant. The integer values are precise.

Reply:
I've deleted these for now; can add back when neede